database/seeders/PermissionsSeeder.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Waivers\Database\Seeders;
+
+use Carbon\Carbon;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\Schema;
+
+class PermissionsSeeder extends Seeder
+{
+    public function run()
+    {
+        if (Schema::hasTable('permissions')) {
+            $now = Carbon::now()->format('Y-m-d H:i:s');
+            $permissions = [
+                'view signatures',
+                'create signatures',
+                'update signatures',
+                'delete signatures'
+            ];
+
+            DB::table('permissions')->insertOrIgnore(
```

Comment:
I would avoid calling database directly, cant we use `app('permission')` or something?

Reply:
Permissions is not our model.

Reply:
We could extend it from that Spatie package and have it be our own model in the tger/authorization package

Reply:
Not necessary.   Once the spatie package is a dependency, the seeder implementation is entirely different.  I'll illustrate in a PR on vouchers.

Reply:
https://github.com/tipoff/vouchers/pull/18